index3.html

```diff
@@ -0,0 +1,285 @@
+          <a class="btn btn-default fullscreen" style="opacity:0.3;position:absolute;z-index:3;margin:6px;"><i class="fa fa-expand"></i></a>
+          <div onClick="$('.choose-prompt').hide();" class="choose-prompt" style="display:inline;color:white;position:absolute;z-index:3;padding-top:14px;text-align:left;padding-left:60px;">
+            <p>1. Choose an input above: select an image, or choose live video.</p>
+            <p>2. Choose an analysis mode or click PRESETS.</p>
+            <p>3. Colorize your results (optional).</p>
+            <p>Click this message to hide it.</p>
+          </div>
+          <div style="background:#111;display:none;position:absolute;top:0;left:0;z-index:2;width:100%;height:100%;" id="backdrop"></div>
+          <div id="overlay-container" class="container" style="display:none;">
+            <img id="overlay-img" src="assets/grid.svg">
+          </div>
+          <a class="btn-inverse btn-small" style="display:none;position:absolute;top:6px;left:6px;z-index:3;" id="exit-fullscreen"><i class="fa fa-remove"></i> Exit full screen</a>
+          <canvas class="fullscreen" id="image" width="800" height="600"></canvas>
+          <div id="colorbar-container" style="display:none;">
+            <span id="colorbar-min">-1.00</span>
+            <canvas id="colorbar"></canvas>
+            <span id="colorbar-max">1.00</span>
+          </div>
+
+        </div>
+
+        <hr style="clear:both;">
+
+        <h2>Q&A</h2>
+
+        <p>Ask and answer questions to peer-troubleshoot this system:</p>
+
+        <iframe style="border:none;" width="100%" height="900px" src="//publiclab.org/embed/grid/question:infragram?"></iframe>
+
+      </div>
+
+      <footer class="footer">
+        <hr>
+        <h4 style="margin-bottom:40px;"><a style="color:#444;" href="http://publiclab.org/infragram" target="_blank">Infragram</a> <small>by <a href="http://publiclab.org" target="_blank">Public Lab</a> | This web app is <a href="https://github.com/publiclab/infragram/" target="_blank">open source software</a>; <a href="https://github.com/publiclab/infragram/issues" target="_blank">please report bugs</a></small></h4>
+        <p>Infragram was created by <a href="https://github.com/bgamari" target="_blank">Ben Gamari</a>, <a href="http://github.com/donblair" target="_blank">Don Blair</a>, <a href="http://publiclab.org/profile/warren">Jeff Warren</a> and <a href="http://github.com/pfoltyn" target="_blank">Piotr Foltyn</a> and others from <a href="http://publiclab.org/contributors/infragram" target="_blank">Public Lab</a>.</p>
+      </footer>
+
+    </div>
+
+    <script>
+      //tooltip hides after one click
+      $('[rel="tooltip"]').tooltip({
+        trigger: "hover",
+      });
+      var infragram, openInPl, openInSequencer;
+      (function() {
+        infragram = Infragram({
+          uploader: false,
+          processor: 'webgl'
+        });
+
+        openInPl = function openInPl() {
```

Comment:
I guess we should move this into a shareable .js file or into the library itself... Just an idea?

Reply:
Please help me with more details on this file. I can't seem to find a file named sharable.js in the infragram directory , I am somehow confused on the mentioned library. Please  some detail on this idea.

Reply:
Sorry I'm typing from phone, I just meant a separate js file we can include into the different html files... A shareable file. Thanks!

Reply:
Okay. Noted

Reply:
thank you